tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.pytest_customization.marks import (
+    magenta_squad,
+    skipif_ocs_version,
+    skipif_ocp_version,
+    bugzilla,
+)
+from ocs_ci.framework.testlib import E2ETest, tier2
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def run_metadata_io_with_cephfs(dc_pod_factory):
+    """
+    This function facilitates
+    1. Create PVC with Cephfs, access mode RWX
+    2. Create dc pod with Fedora image
+    3. Copy helper_scripts/meta_data_io.py to Fedora dc pod
+    4. Run meta_data_io.py on fedora pod
+
+    """
+    access_mode = constants.ACCESS_MODE_RWX
+    file = constants.METAIO
+    interface = constants.CEPHFILESYSTEM
+    active_mds_node = cluster.get_active_mds_info()["node_name"]
+    sr_mds_node = cluster.get_mds_standby_replay_info()["node_name"]
+    worker_nodes = get_worker_nodes()
+    target_node = []
+    ceph_health_check()
+    for node in worker_nodes:
+        if (node != active_mds_node) and (node != sr_mds_node):
+            target_node.append(node)
+    for dc_pod in range(4):
+        log.info("Create fedora dc pod")
+        pod_obj = dc_pod_factory(
+            size="30",
+            access_mode=access_mode,
+            interface=interface,
+            node_name=target_node[0],
+        )
+        log.info("Copy meta_data_io.py to fedora pod ")
+        cmd = f"oc cp {file} {pod_obj.namespace}/{pod_obj.name}:/"
+        helpers.run_cmd(cmd=cmd)
+        log.info("meta_data_io.py copied successfully ")
+        log.info("Run meta data IO on fedora pod ")
+        metaio_executor = ThreadPoolExecutor(max_workers=1)
+        metaio_executor.submit(
+            pod_obj.exec_sh_cmd_on_pod, command="python3 meta_data_io.py"
+        )
+
+
+@tier2
+@bugzilla("2141422")
+@magenta_squad
```

Comment:
this is not part of the magenta squad, please add a relevant marker

Reply:
done